src/jsdoc.ts

```diff
@@ -37,44 +37,66 @@ export interface Tag {
  * The public Closure docs don't list all the tags it allows; this list comes
  * from the compiler source itself.
  * https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/parsing/Annotation.java
+ * https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/parsing/ParserConfig.properties
```

Comment:
As discussed, I've reverted to just the ones we know are load bearing and probably something we want to emit.